karapace/schema_registry_apis.py

```diff
@@ -52,6 +53,7 @@ class SchemaErrorCodes(Enum):
     INVALID_SUBJECT = 42208
     SCHEMA_TOO_LARGE_ERROR_CODE = 42209
     NO_MASTER_ERROR = 50003
+    KAFKA_TIMEOUT_ERROR_CODE = 50004
```

Comment:
Better error code would be `50002` which is `OPERATION_TIMEOUT_ERROR_CODE` in CSR.
`50004` is reserved for unknown leader error.